src/base.ts

```diff
@@ -7,11 +7,11 @@ import {
   cleanUndefinedProperties,
 } from "./utils";
 import HTTP, { HttpResponse } from "./http";
-import endpoint from "./endpoint";
+import Endpoints from "./endpoints";
```

Comment:
small nitpick: why did you switch from lowercase `endpoints` to uppercase `Endpoints`?